esmvaltool/cmorizers/obs/cmorize_obs_grace.py

```diff
@@ -0,0 +1,175 @@
+"""ESMValTool CMORizer for GRACE.
+
+Tier
+   Tier 3
+Source
+   https://podaac.jpl.nasa.gov/dataset/TELLUS_GRAC-GRFO_MASCON_CRI_GRID_RL06_V2
+Last access
+   20200612
+
+Download and processing instructions
+ - Go to the above link
+ - Click the tab "Data Access"
+ - Log in with Earthdata account
+ - Download the following files:
+     - CLM4.SCALE_FACTOR.JPL.MSCNv02CRI.nc
+     - GRCTellus.JPL.200204_202004.GLO.RL06M.MSCNv02CRI.nc
+     - LAND_MASK.CRI.nc
+ - Download the grace months table  which holds important information
+   on data coverage. Save it in the RAWOBSDIR.
+      https://podaac-tools.jpl.nasa.gov/drive/files/allData/tellus/L3/docs/GRACE_GRACE-FO_Months_RL06.csv
+ - Manually inspect and check the months table
+
+
+Modification history
+   20200630-crezee_bas: written.
+"""
+
+import logging
+import os
+from copy import deepcopy
+from datetime import datetime
+
+import iris
+import numpy as np
+import pandas as pd
+import xarray as xr
+from cf_units import Unit
+from dateutil import relativedelta
+
+from esmvalcore.preprocessor import regrid_time
+from esmvaltool.cmorizers.obs import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _make_monthly_data_contiguous(in_file, out_file, raw_varname, cfg):
+    original = xr.open_dataset(in_file)[raw_varname]
+
+    months_table_file = os.path.join(cfg['in_dir'],
+                                     'GRACE_GRACE-FO_Months_RL06.csv')
+    grace_months_table = pd.read_csv(months_table_file)
```

Comment:
Perhaps better not to hardcode the filenames but use the config file: GRACE.yml?